packages/node_modules/@webex/webex-core/test/unit/spec/services/interceptors/server-error.js

```diff
@@ -0,0 +1,161 @@
+import chai from 'chai';
```

Comment:
This is the refactored unit tests after the migration. Note that interceptors cannot be integration tests due to how they exist in unreferencable memory after being mounted.